core-services/prow/03_deployment/deck.yaml

```diff
@@ -284,11 +284,16 @@ objects:
           - -cookie-secret-file=/etc/proxy/secrets/session_secret
           - -tls-cert=/etc/tls/private/tls.crt
           - -tls-key=/etc/tls/private/tls.key
+          - --hidden-only=true
+          - --gcs-credentials-file=/etc/gce-sa-credentials-gcs-private/service-account.json
```

Comment:
This does not seem correct - these are args for the `oauth-proxy`, not for deck.